feedback/feedbackblock.py

```diff
@@ -56,6 +56,12 @@ class FeedbackXBlock(XBlock):
         help="Label for button exit.",
     )
 
+    end_course_label = String(
```

Comment:
and why the `end_course` prefix? ending the course is the purpose of this XBlock.